apps/docs/layouts/guides/index.tsx

```diff
@@ -65,12 +65,19 @@ const Layout: FC<Props> = (props) => {
   return (
     <>
       <NextSeo
-        title={`${props.meta?.title} | Supabase`}
+        title={`${props.meta?.title} | Supabase docs`}
```

Comment:
For consistency with `apps/docs/layouts/DefaultLayout.tsx`
```suggestion
        title={`${props.meta?.title} | Supabase Docs`}
```